docs/media/how-to-guides/how-to-enable-streaming-mode/scripts/chat_app.py

```diff
@@ -6,7 +6,7 @@
 from azure.identity import AzureCliCredential
 import logging
 from azure.ai.ml import MLClient
-from event_stream import EventStream
+from sseclient import SSEClient 
```

Comment:
Maybe we need to add a requirements.txt to list up these external packages.

Reply:
Good idea, I'll add a requirements.txt for these dependencies.